### ISSUE TITLE
Autoload names are incorrect in Project Settings under Filesystem

### ISSUE THREAD

**Issue by MaaaxiKing** [godotengine/godot#39639]
**Godot version:**
3.2.1.stable.official
**OS/device including version:**
Windows 10 Home 1909
**Issue description:**
I have an autoloaded scene and an autoloaded script. At Project --> Project Settings --> Filesystem --> Autoload the names I've given the autoload singeltons are different.
![Screenshot (70)](https://user-images.githubusercontent.com/60623532/85007114-5e45e280-b15b-11ea-8297-c22c02982716.png)
![Screenshot (69)](https://user-images.githubusercontent.com/60623532/85007140-64d45a00-b15b-11ea-97da-7dc1f44d0b24.png)
**Steps to reproduce:**

**Minimal reproduction project:**

**Comment by clayjohn** [comment]
To me, that just looks like a formatting difference. Under Filesystem > Autoload it is just coverting CamelCase into english sentence case. 

Is this causing any issues other than just looking wrong?

**Comment by MaaaxiKing** [comment]
No, no real **problems**.

**Comment by akien-mga** [comment]
The autoformatting should be skipped for the Autoload settings to prevent the confusion.

**Comment by dreamsComeTrue** [comment]
I do know that might be not a proper answer - but for `master` branch - there is no such section as `FileSystem` and additional `AutoLoad` list in it in Project Settings.

So that fix surely should be done only on `3.2` branch.

**Comment by akien-mga** [comment]
I don't think that this changed between 3.2 and master, but you need to define autoloads in the project settings autoloads tab for them to be added to project.godot (and thus the general tab).
